File: DAY-7/Q4.c
#include<stdio.h>

int Reverse(int n);

int main(){
    int n;
    printf("Enter Number:");
    scanf("%d",&n);
    printf("Reverse Of The Given Number is: %d",Reverse(n));
    return 0;
}

int rev = 0;
int Reverse(int n){
    if(n==0){
        return rev;
    }
    else{
        rev = rev*10+(n%10);
        return Reverse(n/10);
    }
}

File: DAY-11/Q4.java
import java.util.Scanner;

public class Q4 {
    public static int Fac(int n){
        int fac=1;
        if(n==0){
            return 1;
        }
        else{
          for(int i=1;i<=n;i++){
            fac=fac*i;
          }
        }
     return fac;
    }


    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter Number: ");
        int n = sc.nextInt();
        int result = Fac(n);
        System.out.print("The Factorial Of Given Number = " +result);
    }
}
